numpy/distutils/fcompiler/__init__.py

```diff
@@ -472,10 +474,8 @@ def customize(self, dist = None):
         fixflags = []
 
         if f77:
-            f77 = _shell_utils.NativeParser.split(f77)
             f77flags = self.flag_vars.f77
```

Comment:
Some context - the root cause here is that `f77` could either be initialized by:
* `some_legal_f77_command[0]`, in which case it represents a single string
* `os.environ["SOME_ENV_VAR"]`, in which case it represents a string that needs splitting according to shell rules
* some other things probably too